demo/articles/arabic.html

```diff
@@ -0,0 +1,64 @@
+                        <div class="outer">
+                            <img alt="Image/" src="../img/oval.png" class="oval" />
+                        </div>
+                        <div class="inner">
+                            <img alt="Image/" src="../img/fill1.png" class="fill1" />
+                            <img alt="Image/" src="../img/fill2.png" class="fill2" />
+                        </div>
+                    </div>
+                </a>
+            </div>
+            <div class="title">
+                <p>أم كلثوم (مطربة)</p>
+            </div>
+            <div class="content">
+                <p>
+                    فاطمة بنت الشيخ المؤذن إبراهيم السيد البلتاجي وتعرف أيضاً بكُنيتها المشهورة أُم كَلثوم، كما تُعرف بعدة ألقاب منها: ثومة، الجامعة العربية، الست، سيدة الغناء العربي، شمس الأصيل، صاحبة العصمة، كوكب الشرق، قيثارة الشرق، فنانة الشعبهي مغنية وممثلة مصرية، ولدت في محافظة الدقهلية بالخديوية المصرية في 31 ديسمبر 1898 أو رسميًا حسب السجلات المدنية في 4 مايو 1908م، وتوفيت في <span class="wmf-wp-with-preview" data-wikipedia-preview>القاهرة</span> بعد معاناة مع المرض في 3 فبراير 1975م. وتعد أم كلثوم من أبرز مغني القرن العشرين الميلادي، وبدأت مشوارها الفني في سن الطفولة، اشتهرت في <span class="wmf-wp-with-preview" data-wikipedia-preview>مصر</span> وفي عموم الوطن العربي
+                </p>
+                <p>
+                    عادت إلى القاهرة لكي تستقر نهائياً في عام 1921م. وكانت تغني في مسرح البوسفور في ميدان رمسيس بدون فرقة موسيقية. وغنت على مسرح حديقة الأزيكية واشتهرت بقصيدة وحقك أنت المنى والطلب وتعد هذه أول إسطوانة لها صدرت في منتصف العشرينيات وبيع منها ثمانية عشر ألف إسطوانة. وتعلمت أم كلثوم لاحقاً من أمين المهدي أصول الموسيقى، وكذلك تعلمت عزف العود على يدي أمين المهدي ومحمود رحمي ومحمد القصبحي.
+في عام 1923م غنت في حفلات كبار القوم، كما غنت في حفل حضرته كبار مطربات عصرها وعلى رأسهم منيرة المهدية شخصياً والتي كانت تلقب بسلطانة الطرب، والتقت في نفس العام بالموسيقار <span class="wmf-wp-with-preview" data-wikipedia-preview>محمد_عبد_الوهاب</span> لأول مرة بحفلة أقيمت في منزل والد أبي بكر خيرت.
+في عام 1924م تعرفت إلى أحمد رامي عن طريق أبو العلا، في إحدى الحفلات التي أدت أم كلثوم فيها أغنية الصب تفضحه عيونه كان أحمد رامي حاضراً بعد أن عاد من أوروبا، فأدرك أنه قد وجد هدفه. غير أن البداية الحقيقية كانت عندما سمعها محمد القصبجي الملحن المجدد وقتها. في نفس العام 1924م، تعرفت أم كلثوم إلى طبيب أسنان يهوى الموسيقى هو أحمد صبري النجريدي أول ملحن يلحن لأم كلثوم ألحانا خاصة بها، إلا أن ألحانه اعتمدت على الزخارف الموسيقية بشكل مبالغ به مما دفع أم كلثوم إلى إنهاء التعاون معه مبكراً.
+                </p>
+
+            </div>
+        </div>
+        <div class="footer">
+            <p>
+                This article uses material from the Wikipedia article “<a href="https://ar.wikipedia.org/wiki/%D8%A3%D9%85_%D9%83%D9%84%D8%AB%D9%88%D9%85_(%D9%85%D8%B7%D8%B1%D8%A8%D8%A9)" target="_blank">הירח</a>",<br />which is released under the Creative Commons Attribution-Share-Alike License 3.0.
```

Comment:
This link is not supposed to have a preview on it. 

When you call `init`, you can specify which element to consider instead of the whole page.

The text of the `<a>` tag is Hebrew.

Reply:
That's three strikes for me! 

Removing the `detectLinks` flag from the arabic article as it's not needed anymore. 

All three footers fixed with: https://github.com/wikimedia/wikipedia-preview/pull/74/commits/27e3d3f1eade115e62d0632ce11a6da513d2419a